flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/factory/MySqlDataSourceFactory.java

```diff
@@ -128,6 +131,8 @@ public DataSource createDataSource(Context context) {
         int connectMaxRetries = config.get(CONNECT_MAX_RETRIES);
         int connectionPoolSize = config.get(CONNECTION_POOL_SIZE);
         boolean scanNewlyAddedTableEnabled = config.get(SCAN_NEWLY_ADDED_TABLE_ENABLED);
+        boolean scanIncrementalNewlyAddedTableEnabled =
```

Comment:
```suggestion
        boolean scanInBinlogPhaseNewlyAddedTableEnabled =
```

Reply:
Which version does it support? How to set it up?